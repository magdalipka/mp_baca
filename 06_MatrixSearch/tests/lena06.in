1
3 7
0 0 1 2 3 4 4
1 2 2 3 3 4 4
2 2 4 6 6 6 6
4

