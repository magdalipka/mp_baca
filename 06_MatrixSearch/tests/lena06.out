RekPier: 4 w (0,5)
RekOst: 4 w (2,2)
IterPier: 4 w (0,5)
IterOst: 4 w (2,2)
---
